libraries-2/src/test/java/com/baeldung/jcommander/usagebilling/cli/SubmitUsageCommandUnitTest.java

```diff
@@ -0,0 +1,60 @@
+package com.baeldung.jcommander.usagebilling.cli;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParameterException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class SubmitUsageCommandUnitTest {
+
+    private static JCommander jc;
+
+    @BeforeClass
+    public static void setup() {
```

Comment:
I would recommend moving this to instance level with an initializer. As much as you can argue that the JCommander object is not tiny and is immutable, `static` test setup of anything other than constant Strings is a special case and it's not needed here.

Reply:
makes sense, fixed.